src/pages/test-report/components/manoeuvre-competency/manoeuvre-competency.ts

```diff
@@ -34,7 +34,7 @@ interface ManoeuvreCompetencyComponentState {
   selector: 'manoeuvre-competency',
   templateUrl: 'manoeuvre-competency.html',
 })
-export class ManoeuvreCompetencyComponent implements OnInit, OnDestroy {
+export class ManoeuvreCompetencyComponent implements OnInit {
```

Comment:
See you've removed declaration that this component implements `OnDestroy`, but we still use that method. What's the reason for this?

Reply:
This has just been omitted by mistake. Added back in on last commits